File: cryze_android_app/android/app/src/main/java/com/tencentcs/iotvideo/netconfig/NetConfigResult.java
package com.tencentcs.iotvideo.netconfig;

/* loaded from: classes2.dex */
public class NetConfigResult {
    public static final int STATUS_FAILED = 0;
    public static final int STATUS_WAITING = 1;
    private String devId;
    private int status;
    private String token;

    public String getDevId() {
        return this.devId;
    }

    public int getStatus() {
        return this.status;
    }

    public String getToken() {
        return this.token;
    }

    public void setDevId(String str) {
        this.devId = str;
    }

    public void setStatus(int i10) {
        this.status = i10;
    }

    public void setToken(String str) {
        this.token = str;
    }

    public String toString() {
        StringBuilder sb2 = new StringBuilder("NetConfigResult{token='");
        sb2.append(this.token);
        sb2.append("', devId='");
        sb2.append(this.devId);
        sb2.append("', status=");
        sb2.append(this.status);
        sb2.append('}');
        return sb2.toString();
    }
}

File: cryze_android_app/android/app/src/main/java/com/tencentcs/iotvideo/netconfig/NetConfigResult.kt
package com.tencentcs.iotvideo.netconfig

import androidx.core.app.NotificationCompat
import com.google.gson.annotations.SerializedName

/* loaded from: classes12.dex */
class NetConfigResult {
    @SerializedName("devId")
    var devId: String? = null

    @SerializedName(NotificationCompat.CATEGORY_STATUS)
    var status: Int = 0

    @SerializedName("token")
    var token: String? = null

    override fun toString(): String {
        return "NetConfigResult{token='" + this.token + "'" + ", devId='" + this.devId + "'" + ", status=" + this.status + "}"
    }

    companion object {
        const val STATUS_FAILED: Int = 0
        const val STATUS_WAITING: Int = 1
    }
}